.circleci/config.yml

```diff
@@ -491,11 +491,27 @@ jobs:
             - dist
 
   # run ember frontend tests
-  ember-test:
+  ember-test-oss:
+    docker:
+      - image: *EMBER_IMAGE
+    environment:
+      EMBER_TEST_REPORT: test-results/report.xml #outputs test report for CircleCI test summary
+    steps:
+      - checkout
+      - restore_cache:
+          key: *YARN_CACHE_KEY
+      - attach_workspace:
+          at: ui-v2
+      - run:
+          working_directory: ui-v2
+          command: make test-ci
```

Comment:
this should be `make test-ci-oss`

Reply:
Oh :doh: great spot Alvin, thanks for that. I just changed this and changed the names of the reports also 👍